install-worker.sh

```diff
@@ -73,6 +77,13 @@ cat <<EOF | sudo tee -a /etc/chrony.conf
 rtcsync
 EOF
 
+# Make tsc the clock source
+if grep --quiet tsc /sys/devices/system/clocksource/clocksource0/available_clocksource; then
+    sudo bash -c "echo tsc > /sys/devices/system/clocksource/clocksource0/current_clocksource"
```

Comment:
You can probably just do
```bash
echo "tsc" | sudo tee /sys/devices/system/clocksource/clocksource0/current_clocksource
```

instead of echoing in a shell